### PR TITLE
Added Periodic Table Elements Information in Python Script

### LINKED ISSUE #781: Periodic Table Elements Information
In this script,
will create an application that will allow user to get the details about the any periodic table elements.

This will include all the 118 element that are in periodic table.

For each periodic element, it will show details about its
1.) symbol
2.) Appearance (how that element looks like)
3.) What is it's atomic number
4.) That element is present in which group block of periodic table
5.) Period of the element
6.) The category of element to which it belongs

And will try to add more details.

### PR DESCRIPTION
**Pull request by akash435**
- A "Periodic Table Elements Information" is an application created in python with tkinter gui.
- In this application, user can find the details about any periodic table elements.
- Here it supports all the 118 elements of the periodic table.
-The details for each mission will be in terms of Name, Symbol, Appearance, Atomic Number, Group Block, Period, Element Category, Atomic Weight, Electronic Configuration, Phase, Melting Point, Boiling Point, Heat of Vaporization, Heat of Fusion, Molar Heat Capacity, Oxidation States, Electronegativity, Ionization Energies, Covalent Radius, Van Der Waal's Radius, Crystal Structure, Thermal Conductivity, Magnetic Ordering.

Fixes #781

## Checklist

- [x] The PR title starts with "Fix #issue_number".  
- [x] My branch is up-to-date with the main branch.
- [x] The PR does not contain any unnecessary auto-generated code.
- [x] The PR does not break any part of the existing code.
- [x] The PR creator have carefully read & adher to the guidelines of the project.
- [x] The PR is raised after the issue has been assigned to the PR creator.

Below are some screenshots showing how application works :

### CODE REVIEW

**Review by GowthamGoush — COMMENTED** [review]
If those commented lines don't convey anything about the script, then please do consider removing it.

File: Python/Periodic Table Elements Information/periodic_table_elements_information.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+oxidation_states = data['oxidation_states'].tolist()
+electronegativity = data['electronegativity'].tolist()
+ionization_energies = data['ionization_energies'].tolist()
+covalent_radius = data['covalent_radius'].tolist()
+van_der_waals_radius = data['van_der_waals_radius'].tolist()
+crystal_structure = data['crystal_structure'].tolist()
+thermal_conductivity = data['thermal_conductivity'].tolist()
+magnetic_ordering = data['magnetic_ordering'].tolist()
+
+# print(len(number))
```
Comment (GowthamGoush) [inline Python/Periodic Table Elements Information/periodic_table_elements_information.py:62]: So, I was mentioning the comments starting from this line.

**Review by GowthamGoush — COMMENTED** [review]
I've mentioned it below.

**Comment by akash435** [comment]
@GowthamGoush , removed the unnecessary comments.

**Review by GowthamGoush — APPROVED** [review]
Ready to be merged.

**Pull request merged**